packages/snap-preact-components/src/components/Organisms/Autocomplete/Autocomplete.tsx

```diff
@@ -170,6 +170,7 @@ export const Autocomplete = observer((properties: AutocompleteProps): JSX.Elemen
 		// default props
 		termsTitle: '',
 		trendingTitle: 'Popular Searches',
+		historyTitle: 'Previously Searched',
```

Comment:
`hideHistory` has a default value of `true` in the stories file however it looks to have a default value of `false` in the component file as it's not defined here.

Reply:
good catch, it was true by default initially, but refactored to undefined.